lua/entities/gmod_wire_expression2/core/chat.lua

```diff
@@ -79,11 +79,11 @@ end
 --- Returns the last player to speak.
 e2function entity lastSpoke()
 	local entry = TextList.last
-	if not entry then return nil end
```

Comment:
I notice you made a lot of these changes. Do these actually cause errors, or does the core code consider `nil` and `NULL` to be the same anyway?